### PR TITLE
Holy water

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Holy water, It heals you like tricordazine but less. It looks lighter than water and can be easily identified through examine. I saw that chaplains could turn blood into wine, and decided to add onto that content because chaplains currently can't do enough right now. The healing isn't strong enough to invalidate tricordazine though since you only heal 1u per sip. 

Currently, it is mostly for roleplay purposes but the real goal of this PR is to add holy water for future chaplain content. Some examples include splashing holy water on revenants to deal high damage, or cleaning impure/corrupted reagents. It could even act as a recipe for future chaplain reagents that could have special abilities.

_**I am going to make a new pr for a new damage damage type and modifier; "Holy", that deals extra damage to unholy entities.**_

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

image

**Changelog**

:cl: Ubaser
- add: Chaplains can smack their bible on water to create holy water.

### CODE REVIEW

**Comment by UbaserB** [comment]
just realised i can make holy water deal damage to unholy entities one second

**Comment by lzk228** [comment]
holy crap! holy water real!

File: Resources/Prototypes/Reagents/medicine.yml [hunk 1]
```diff
@@ -1001,3 +1001,23 @@
         damage:
           types:
             Poison: 4
+
+- type: reagent
+  id: Holy water
```
Comment (lzk228) [inline Resources/Prototypes/Reagents/medicine.yml:1006]: pro thought its a name:trollface:

Comment (UbaserB) [reply Resources/Prototypes/Reagents/medicine.yml:1006]: i have brain rot

**Comment by lzk228** [comment]
[Common] Urist McMedic says: "Chaplain please come in medbay and make holy water for us"
isn't that goin to be like that or something? :trollface:

**Comment by UbaserB** [comment]
> [Common] Urist McMedic says: "Chaplain please come in medbay and make holy water for us" isn't that goin to be like that or something? :trollface:

perhaps, that could totally happen but its still weaker than tricord.

**Comment by JoeHammad1844** [comment]
kult

**Pull request merged**